ApiSurface/Type.fs

```diff
@@ -98,7 +99,10 @@ module internal Type =
             |> sprintf "(%s)"
             |> sprintf
                 "%s%s"
-                (if t.Name.StartsWith "ValueTuple" && t.Namespace = "System" then
+                (if
+                     t.Name.StartsWith ("ValueTuple", false, CultureInfo.InvariantCulture)
```

Comment:
It's [essentially always wrong to use `CultureInfo.InvariantCulture`](https://learn.microsoft.com/en-us/dotnet/standard/base-types/best-practices-strings) in string comparison. The invariant culture is only useful for dealing with data that's not actually a string. The whole thing is a complete mess :( 
```suggestion
                     t.Name.StartsWith ("ValueTuple", StringComparison.Ordinal)
```

Reply:
(It does the right thing here, because the text is ASCII, but using the invariant culture with strings is a big code smell in general.)

Reply:
> Could we consider changing the analyser's warning to include "You probably do not want to use the invariant culture here; read https://learn.microsoft.com/en-us/dotnet/standard/base-types/best-practices-strings before doing so"?

That's a very good idea, thanks!